src/main/java/com/sun/tdk/signaturetest/core/MemberCollectionBuilder.java

```diff
@@ -462,7 +462,7 @@ private void addInheritedFromInterfaces(ClassDescription cl,
                     } else if (memb.isInner()) {
 
                         if (!internalClasses.contains(memb.getName()) && retVal.findSimilar(memb) == null) {
-                            retVal.addMember(memb);
```

Comment:
This is the fix for problems found during [PR-2960](https://github.com/oracle/graal/pull/2960). Is there any reason why "nested classes" list shall be inherited? CCing: @jlahoda, @gradinac

Reply:
Thank you very much for fixing this on such a short notice, we really appreciate it! :)